### DISCUSSION TITLE
Documentation pop-up covers up my cursor

### DISCUSSION THREAD

**Discussion by justjake** [LazyVim/LazyVim#5224] [Q&A]
Just installed LazyVim and trying to massage it a bit. When I'm editing Lua config files, I appreciate getting a nice documentation pop-up, but it almost always covers up the code I'm writing which is really frustrating. It seems to try to avoid covering the horizontal split below? Does anyone have a solution to this issue?

<img width="1003" alt="image" src="https://github.com/user-attachments/assets/483a23a7-7bec-403f-b679-6fe4319dcfc3" />

**Comment by gmnz** [comment]
I use LazyVim daily for work and have never seen this. Try to reinstall from scratch maybe? Check if you have all the required installation dependencies and their right versions and see if it still happens?

 Or maybe you just somehow disabled text wrapping in the floats? 

![image](https://github.com/user-attachments/assets/2c2d97b2-4205-404e-99e7-c2f28f98ef29)


Cause the text wraps in my floats.

**Reply by justjake** [reply]
Can you share your lock file? I wonder if I have a bad version of something, I tried updating a few plugins but no luck. I took the screenshot with basically fresh install - only customization was adding a colorscheme from base16-nvim
